### DISCUSSION TITLE
How to use charts and tables

### DISCUSSION THREAD

**Discussion by Noobert01** [frangoteam/FUXA#919] [General]
Hello all,

I would like to use FUXA to record the temperatures and humidity in my flat. I have selected the history mode but neither the table nor the chart fills up. Do I have to pay attention to anything here?

How long can the history table actually be?

Does the recording actually continue when I switch to the editor? When does the recording start?

Many greetings
Steffen

**Comment by unocelli** [comment]
Hi,
About the first question: https://github.com/frangoteam/FUXA/issues/921
The long of history depend, in any case you make with paging, that help the rendering if you have to many values.
Yes, the recording is a backend (server) jobs
